ovs/extensions/healthcheck/alba/alba_health_check.py

```diff
@@ -586,88 +577,108 @@ def get_disk_safety(logger):
                     applicable_dead_osds = bucket_safety['applicable_dead_osds']
                     # Amount of lost disks at this point
                     bucket[2] = bucket[2] - applicable_dead_osds
-                    disk_lost = bucket[2] - (bucket[0] + bucket[1])
-                    if disk_lost not in disk_lost_overview:
-                        disk_lost_overview[disk_lost] = 0
-                    disk_lost_overview[disk_lost] += objects
-            for disk_lost, objects in disk_lost_overview.iteritems():
+                    backend_overview = bucket[2] - (bucket[0] + bucket[1])
+                    if backend_overview not in disk_lost_overview:
+                        disk_lost_overview[backend_overview] = 0
+                    disk_lost_overview[backend_overview] += objects
+            for backend_overview, objects in disk_lost_overview.iteritems():
                 lost = {
                     'measurement': 'disk_lost',
                     'tags': {
-                        'backend_name': ab.name,
-                        'disk_lost': disk_lost
+                        'backend_name': alba_backend.name,
+                        'disk_lost': backend_overview
                     },
                     'fields': {
                         'total_objects': total_objects,
                         'objects': objects
                     }
                 }
                 points.append(lost)
-
         if len(points) == 0:
             logger.skip('Found no objects present of the system.', test_name)
         else:
-            backends_to_be_repaired = {}
-            for result in points:
+            checked_backends = {
+                "healthy": {},
+                "faulty": {}
+            }
+            for point in points:
```

Comment:
Its completely useless to have these ```points```
Many of these loops can be merged into 1 loop
Imo the only loops that are required are the ```for namespace in namespaces``` loop to collect all information about the backend you passed the config in. And then 1 more loop to parse this information